lib/PuppeteerSharp/Page.cs

```diff
@@ -981,17 +981,17 @@ public Task EmulateMediaAsync(MediaType media)
         /// Closes the page.
         /// </summary>
         /// <returns>Task.</returns>
-        public Task CloseAsync()
+        public async Task CloseAsync()
         {
             if (!(Client?.Connection?.IsClosed ?? true))
             {
-                return Client.Connection.SendAsync("Target.closeTarget", new
+                await Client.Connection.SendAsync("Target.closeTarget", new
```

Comment:
can't we just use return for both tasks? (and in line 993)

Reply:
I don't think so. Both are in the same if block.
```cs 
await Client.Connection.SendAsync();
await Target.CloseTask;
```